data/byte_stream_split.zstd.parquet.md

```diff
@@ -0,0 +1,27 @@
+`byte_stream_split.zstd.parquet` is generated by pyarrow 14.0.2 using the following code:
+
+```python
+import pyarrow as pa
+from pyarrow import parquet as pq
+import numpy as np
+
+np.random.seed(0)
+table = pa.Table.from_pydict({
+  'f32': np.random.normal(size=300).astype(np.float32),
+  'f64': np.random.normal(size=300).astype(np.float64),
+})
+
+f = '/Users/martin/Downloads/byte_stream_split.parquet'
```

Comment:
Just writing into the current directory would make this script more readily usable.

Reply:
Oops - fixed